polymetis/polymetis/src/polymetis_server.cpp

```diff
@@ -253,20 +255,23 @@ Status PolymetisControllerServerImpl::SetController(
 
   try {
     // Load new controller
-    auto new_controller = new TorchScriptedController(
+    new_controller = new TorchScriptedController(
```

Comment:
```cpp
auto new_controller = std::make_unique<TorchScriptedController>(...);
...
custom_controller_context_.custom_controller.reset(std::move(new_controller));
```

Reply:
Changed to use `std::swap`, slightly concerned about having to reference the controller using `std::unique_ptr::get` when choosing a controller to inference in `ControlUpdate`.